### ISSUE TITLE
Feature: Ignore comments in .terraform-version

### ISSUE THREAD

**Issue by lifeofguenter** [tfutils/tfenv#283]
Would be great if the version reader could be expanded slightly beyond a `cat` to allow the ignoring of comments in the `.terraform-version` file.

**Comment by webyneter** [comment]
Yep, if the comments would be ignored I could use Terragrunt to generate it, however, at the moment it's not possible.

**Comment by webyneter** [comment]
Any timeline for this one?
